airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -314,6 +325,24 @@ void testDiscover() throws Exception {
     });
   }
 
+  @Test
+  void testDiscoverWithNonCursorFields() throws Exception {
+    // could not find a way to add specific types that could not be used as cursor fields for mssql, db2
+    // and snowflake
+    if (!(getDriverClass().toLowerCase().contains("microsoft") ||
```

Comment:
how about Set, Enum, Text, and Blob fields in MS SQL - can these be used as cursor?